src/content/docs/ko/concepts/islands.mdx

```diff
@@ -22,11 +23,21 @@ Astro는 **아일랜드**라는 프런트엔드 아키텍처를 개척하고 대
 
 SPA는 [Astro 페이지 내부에도 삽입](/ko/guides/migrate-to-astro/from-create-react-app/)되어 있습니다. 그러나 SPA는 선택적이고 전략적으로 수화하는 기본 능력이 부족하여 오늘날 대부분의 웹 프로젝트에서 과도한 선택이 되고 있습니다.
 
-Astro는 Sylor-Miller가 처음 만든 컴포넌트 아일랜드 패턴을 사용하여 선택적 수화 기능이 내장된 최초의 주류 JavaScript 웹 프레임워크로 인기를 얻었습니다.
+Astro는 Sylor-Miller가 처음 제시한 동일한 컴포넌트 아일랜드 패턴을 사용하여 선택적 하이드레이션이 내장된 최초의 주류 JavaScript 웹 프레임워크로서 인기를 얻었습니다. 우리는 이후 Sylor-Miller의 원래 작업을 확장하고 발전시켰으며, 이는 동적으로 서버에서 렌더링되는 콘텐츠에 대한 유사한 컴포넌트 아일랜드 접근 방식에 영감을 주었습니다.
 
 ## 아일랜드란?
 
-**Astro에서 "아일랜드"는 페이지의 대화형 UI 컴포넌트를 의미합니다.** 아일랜드를 서버 측에서 렌더링된 정적이고 가벼운 HTML이라는 바다 위에 떠있는 대화형 위젯으로 생각하세요.
+Astro에서 아일랜드는 정적 HTML 페이지를 향상시킨 UI 컴포넌트입니다.
+
+[**클라이언트 아일랜드**](#클라이언트-아일랜드)는 페이지의 나머지 부분과 별도로 하이드레이션되는 대화형 JavaScript UI 컴포넌트이며, [**서버 아일랜드**](#서버-아일랜드)는 페이지의 나머지 부분과 별도로 동적 콘텐츠를 서버 렌더링하는 UI 컴포넌트입니다.
+
+두 아일랜드 모두 최적화된 페이지 로드를 위해 컴포넌트별로 비용이 많이 들거나 느린 프로세스를 실행합니다.
+
+## 아일랜드 컴포넌트
+
+Astro components are the building blocks of your page template. They render to static HTML with no client-side runtime.
+
+Think of a client island as an interactive widget floating in a sea of otherwise static, lightweight, server-rendered HTML. Server islands can be added for personalized or dynamic server-rendered elements, such as a logged in visitor's profile picture.
```

Comment:
These lines haven't been translated, could you update them? 🙌

Reply:
@yanthomasdev Thank you!